### ISSUE TITLE
Don't use conda environments

### ISSUE THREAD

**Issue by PestyVesty** [Brikwerk/nxbt#78]
I'm not familiar with Linux package paths, where they are installed etc, which is also a contributing reason why it took me forever to figure out what went wrong.

But as title suggests, don't use conda environment when you are using nxbt. The python Anaconda environment is different to the python version root uses. More specifically, if you try to run code using nxbt in a conda environment, you will get the error ```AF_BLUETOOTH not found```. Apparently Anaconda's python module ```socket``` does not include the attribute ```AF_BLUETOOTH```, while the python root has access to does.

TLDR: deactivate all conda environment before running any code using nxbt module.
